tests/admin.test.ts

```diff
@@ -36,15 +36,18 @@ describe('retroactive attendance submission', () => {
 
     for (let u = 0; u < users.length; u += 1) {
       const user = users[u];
-      const userResponse = await userController.getUser(user.uuid, proxyUser);
+      const userUuid = {
+        uuid: user.uuid,
+      };
+      const userResponse = await userController.getUser(userUuid, proxyUser);
```

Comment:
nit: you can inline a lot of these variable declarations to reduce LoC since they aren't used that often and aren't _that_ cryptic when inlined:
```js
const userResponse = await userController.getUser({ uuid: user.uuid }, proxyUser)
```